src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = true;
```

Comment:
```suggestion
    this.hidden = !this.hidden;
```
Як написано в описі задачі:
> Herbivore has a method of hide, which changes the hidden property of the beast to the opposite 

Зараз цей метод в тебе ставить постійно значення true, а повинен міняти на протилежне